WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/BottomButtonView.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+struct BottomButtonView<Style>: View where Style: ButtonStyle {
+    let style: Style
+    let title: String
+    let image: UIImage?
+    let onButtonTapped: () -> Void
+
+    var body: some View {
+        VStack(spacing: 0) {
+            Divider()
+            Button(action: { onButtonTapped() }) {
+                HStack {
+                    if let image = image {
+                        Image(uiImage: image)
+                    }
+                    Text(title)
+                    Spacer()
+                }
+            }
+            .buttonStyle(style)
+        }.edgesIgnoringSafeArea([.bottom])
```

Comment:
I think `edgesIgnoringSafeArea` is a bit case-specific, i.e in some other screens we may not want to show the button in non-safe area. How about moving it to the package list screen instead?

Also, I know that I've been using  `edgesIgnoringSafeArea` here and there but maybe we should use the newer variant [`ignoresSafeArea(_:edges:)`](https://developer.apple.com/documentation/swiftui/view/ignoressafearea(_:edges:)) since the old one may be deprecated soon 🤔

Reply:
Good point. I just played around with this again and realized that we don't need it even in this case, so I'll remove it entirely. (I thought I'd need it to extend the background to the edge, but I just tried again and the background extends even without it.)

Thanks for the tip about `ignoresSafeArea(_:edges:)` — I missed that it's the newer way of doing this!